builds/service-onboarding-build-pack/Jenkinsfile

```diff
@@ -452,3 +452,38 @@ def jsonParse(jsonString) {
     def nonLazyMap = new groovy.json.JsonSlurperClassic().parseText(jsonString)
     return nonLazyMap
 }
+
+def updateAclPermission(serviceId, userId) {
+	try {
+		def categoryList = ['code', 'deploy']
+		def policiesList = []
+		for (category in categoryList) {
+			def eachPolicy = [
+				userId: userId,
+				permission: 'admin',
```

Comment:
for code and deploy its not admin but write
Manage - read|write|admin
Code - read | write
Deploy - read|write

Reply:
Updated